src/index.js

```diff
@@ -379,6 +391,48 @@ class WhatsAppAPI {
         if (!url) throw new Error("URL must be specified");
         return api.authenticatedRequest(this.token, url);
     }
+
+    /**
+     * POST helper, must be called inside the post function of your code.
+     * When setting up the webhook, only subscribe to messages. Other subscritions support might be added later.
+     *
+     * @param {Object} request The request object sent by Whatsapp
+     * @returns {Number} 200, it's the expected http/s response code
+     * @throws {Number} 400 if the POST request isn't valid
+     */
+    post(request) {
+        //Validating payload
+        const signature = request.header("X-Hub-Signature-256");
+        if (!signature) throw 400;
+
+        const hash = createHmac("sha256", this.appSecret)
+            .update(request.rawBody)
+            .digest("hex");
+        if (signature.split("=")[1] !== hash) throw 400;
+
+        return post(
+            request.body,
+            (...message) => {
+                this.emit('message', ...message);
+            },
+            (...status) => {
+                this.emit('status', ...status);
+            })
```

Comment:
post could be rewritten to just recieve the emitter and simplify the syntax, but I can handle that in a later PR.

What do you think?